cartographer_ros/cartographer_ros/msg_conversion_test.cc

```diff
@@ -82,6 +92,50 @@ TEST(MsgConversion, LaserScanToPointCloudWithInfinityAndNaN) {
       point_cloud[1].isApprox(Eigen::Vector4f(-3.f, 0.f, 0.f, 0.f), 1e-6));
 }
 
+::testing::Matcher<const LandmarkObservation&> EqualsLandmark(
+    const LandmarkObservation& expected) {
+  return ::testing::AllOf(
+      Field(&LandmarkObservation::id, expected.id),
+      Field(&LandmarkObservation::landmark_to_tracking_transform,
+            ::cartographer::transform::IsNearly(
+                expected.landmark_to_tracking_transform, 1e-2)),
```

Comment:
I see no reason why this can be 1% off. Replace all 1e-6, 1e-2 and 0.01 by kEps = 1e-6?